spec/requests/api/v2/chapters_controller_spec.rb

```diff
@@ -8,4 +8,13 @@
       end
     end
   end
+
+  describe 'GET #headings' do
+    let(:heading) { create :heading, :with_chapter }
+    let(:chapter) { heading.reload.chapter }
```

Comment:
Yes, otherwise it is `nil`. We do this in other places, probably we can fix that by changing the factory. Something to improve, but maybe not just now.

Reply:
Cool :+1:

Reply:
You'll actually want to rebase main to get a filename check test rolling